app/controllers/config_reports_controller.rb

```diff
@@ -7,6 +7,7 @@ class ConfigReportsController < ApplicationController
   def index
     respond_to do |format|
       format.html do
+        @host = resource_finder(Host.authorized(:view_hosts), params[:host_id]) if params[:host_id]
```

Comment:
similar issue here (with back btn) 
![image](https://user-images.githubusercontent.com/20167/37873616-ebf64130-3028-11e8-8909-0b3674f5028a.png)

but please note the difference, the breadcrumbs links points to: /config_reports while the back btn points to /config_reports?search=eventful+%3D+true

please remove the back btn and change the url for the index. -thanks